tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t5.py

```diff
@@ -392,7 +392,7 @@
                 Field(
                     item="29",
                     name="AMOUNT_EARNED_INCOME",
-                    friendly_name="amount earnedof income",
+                    friendly_name="Amount of Earned Income",
```

Comment:
this has been fixed. see below (top is original behavior, bottom is the fix) @victoriaatraft

![2952p3](https://github.com/raft-tech/TANF-app/assets/63075587/ce0258c3-d536-42c3-8271-a67c44644282)